.gitignore

```diff
@@ -13,3 +13,5 @@ eslint-junit.xml
 *-junit.xml
 .DS_Store
 tsconfig.tsbuildinfo
+dump.rdb
```

Comment:
Ah did you try running `redis-cli` from the recorder repo path? Not sure if we need this here anyway.

Reply:
It's possible but typically not. I'll remove it anyway as I've never needed it before, probably something weird I did.

Reply:
Fixed c7782d57f90684101fb7fa16e4f98b510fad26f6